### PR TITLE
request to merge assignment 1 into develop branch

### PR DESCRIPTION
**Pull request by munendra-successive**
1.Create a functional component named Greeting that displays a simple "Hello, React!" message on the screen.
Import and render the Greeting component in the App component.

2.Create a functional component named UserCard that accepts the following props: name, email, and avatarURL.
Use the props to display the user's name, email, and an image (avatar) in the UserCard component.
Create multiple instances of the UserCard component with different user data.

3.Create a functional component named Weather that accepts a prop called temperature (a number).
Display a message like "It's sunny today!" if the temperature is above 25°C and "It's cold today!" if the temperature is below 10°C.
Import and render the Weather component in the App component with different temperature values.

4.Create a functional component named Counter that displays a count and two buttons: "Increment" and "Decrement".
Implement event handlers for the "Increment" and "Decrement" buttons to increase and decrease the count.
Display the updated count on the screen.

5.Create a functional component named TaskList that accepts an array of task names as a prop.
Use the map function to render each task name as a list item.
Import and render the TaskList component in the App component with an array of tasks.

6.Create a functional component named Button that accepts props for text and color.
Style the button using inline styles or CSS classes based on the color prop.
Import and render the Button component in the App component with different text and color props.

### CODE REVIEW

File: assignment-1/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
```
Comment (UrvashiChaudhary) [inline assignment-1/package.json:1]: package.json, public directory, node_modules, gitignore, package-lock.json should be in parent directory, not in assignment1

File: assignment-1/.gitignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
```
Comment (UrvashiChaudhary) [inline assignment-1/.gitignore:7]: add package-lock.json in .gitignore

File: assignment-1/public/robots.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```
Comment (UrvashiChaudhary) [inline assignment-1/public/robots.txt:1]: Delete this file if not required

File: assignment-1/src/App.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
```
Comment (UrvashiChaudhary) [inline assignment-1/src/App.css:3]: Remove this file if not required

File: assignment-1/src/logo.svg
Comment (UrvashiChaudhary) [inline assignment-1/src/logo.svg:1]: Please remove it if you're not using it.

**Review by UrvashiChaudhary — CHANGES_REQUESTED** [review]

File: assignment-1/package.json (hunk 1 above)
Comment (munendra-successive) [reply assignment-1/package.json]: Changed the directory of package.json, public directory, node_modules, gitignore, and package-lock.json to parent

File: assignment-1/.gitignore (hunk 2 above)
Comment (munendra-successive) [reply assignment-1/.gitignore]: Added package-lock.json in .gitignore

File: assignment-1/public/robots.txt (hunk 3 above)
Comment (munendra-successive) [reply assignment-1/public/robots.txt]: Deleted robots.txt

File: assignment-1/src/App.css (hunk 4 above)
Comment (munendra-successive) [reply assignment-1/src/App.css]: Deleted App.css

File: assignment-1/src/logo.svg
Comment (munendra-successive) [reply assignment-1/src/logo.svg:1]: removed logo.svg

**Pull request merged**